python/tk_multi_loader/dialog.py

```diff
@@ -298,10 +298,6 @@ def __init__(self, action_manager, parent=None):
 
         #################################################
         # set up cog button actions
-        self._help_action = QtGui.QAction("Show Help Screen", self)
-        self._help_action.triggered.connect(self.show_help_popup)
-        self.ui.cog_button.addAction(self._help_action)
```

Comment:
Does that mean we are no longer able to display the "help screen" pass the "first time"?

Reply:
I thought that was the goal of the ticket. This part removes the button to show it again.